dist/scss/wds-components/_dropdowns.scss

```diff
@@ -17,30 +16,190 @@
 
 	&__content {
 		background-color: $wds-color-white;
-		box-shadow: 0 0 10px 0 rgba($wds-color-black, .2);
+		border: $wds-dropdown-border-width solid $wds-color-faint-gray;
+		border-radius: $wds-dropdown-border-radius;
 		display: none;
-		left: 0;
+		left: 50%;
+		// it's for long dropdowns toggles, to make sure dropdown content is always below the nodge
+		min-width: 70%;
+		// we assume that there will be regular size .wds-list inside the dropdown content
+		padding: $wds-dropdown-padding - $wds-list-item-vertical-padding 0;
 		position: absolute;
-		top: calc(100% + 1px);
+		top: 100%;
+		transform: translateX(-50%);
 		z-index: $content-z-index;
 
-		&.wds-is-dark {
-			box-shadow: 0 0 20px 0 rgba($wds-color-black, .3);
-		}
-
 		&.wds-is-right-aligned {
 			left: auto;
 			right: 0;
+			transform: none;
+		}
+
+		&.wds-is-left-aligned {
+			left: 0;
+			transform: none;
+		}
+
+		// The default is to show scroll on long dropdowns
+		&:not(.wds-is-not-scrollable) {
+			.wds-list {
+				@include scrollable-list(270px);
+			}
+		}
+
+		&.wds-is-not-scrollable {
+			.wds-list {
+				position: relative;
+			}
+		}
+
+		.wds-list {
+			padding: 0;
+
+			> li {
+				margin: 0 $wds-dropdown-padding / 2;
+				padding-left: $wds-dropdown-padding / 2;
+				padding-right: $wds-dropdown-padding / 2;
+				white-space: nowrap;
+			}
+
+			&.wds-is-linked {
+				> li {
+					padding: 0;
+
+					> a {
+						color: $wds-color-dark-blue-gray;
+						padding-left: $wds-dropdown-padding / 2;
+						padding-right: $wds-dropdown-padding / 2;
+						text-decoration: none;
+						transition-duration: $link-transition-duration;
+						transition-property: background-color, color;
+					}
+
+					&:hover > a {
+						@include wds-dropdown-linked-list-item-hover;
+					}
+				}
+			}
 		}
 	}
 
-	&.wds-is-active .wds-dropdown {
-		&__content {
+	&::after {
```

Comment:
What do you think about?
&::after, &::before {
//'common' properties

&::hover {
display: block;
}
}

&::before {
custom before prop
}

&::after {
}